CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Release 2024-07-09
 
+### Amazon.Extensions.CognitoAuthentication (2.5.6)
```

Comment:
We have a new automated process for updating change. Please refer https://github.com/aws/aws-sdk-net-extensions-cognito/blob/master/CONTRIBUTING.md#adding-a-change-file-to-your-contribution-branch. You would need to add a change file using `autover` command along with parameter `--increment-type Patch` (default is `Patch`, so you can omit this parameter).

As part of release process, this changefile would be consumed which will:
- Version-bump this package
- Include changes to CHANGELOG.